src/parakeet/Driver.cpp

```diff
@@ -56,6 +56,11 @@ namespace parakeet
             return false;
         }
 
+        g_withIntensity = intensity;
```

Comment:
Not a fan of the use 'g_' at the start of the variable names here. Also, in light of the book club conversations, it might be worthwhile to clean up this code a little as it's the most complex and most important one in this project.

The \brief for the connect method in the header is also wrong.

Reply:
Good catch on the \brief.

There are several things I'd like to do to the Driver class based off the bookclub (having a struct for parameter is a big one). Should the scope of this PR to match all those changes?

Reply:
Hmmm... you being up a good point. Let's keep these changes scoped to just addressing the units issue and add a YT issue for general clean up which we can work on soon.

Reply:
I modified the \brief to be correct, created YT Issue PRKT-82 to handle the rest of the comments.